.env

```diff
@@ -28,3 +28,5 @@ TEST_PRESTO_HOST=localhost
 TEST_PRESTO_PORT=8080
 TEST_REDIS_HOST=127.0.0.1
 TEST_REDIS_PORT=6379
+TEST_REDIS_OLD_HOST=127.0.0.1
+TEST_REDIS_OLD_PORT=6380
```

Comment:
Nice! I was not entirely sure if this was enough to get docker-compose to assign an alternative port when starting the containers but it is, good find! :)